requirements.in

```diff
@@ -0,0 +1,6 @@
+ipython
+Django
+psycopg2
+django-environ
+sentry_sdk
```

Comment:
and if I go full OCD on this, I would avoid underscores and use hyphens for package names, as they are the preferred convention for pypi package names, at least I think 😅

Reply:
Nowhere near OCD IMO, fixing